src/utils/managing.officer.individual.ts

```diff
@@ -114,10 +114,12 @@ export const removeManagingOfficer = async (req: Request, res: Response, next: N
   }
 };
 
-const setOfficerData = (reqBody: any, id: string): ApplicationDataType => {
+export const setOfficerData = (reqBody: any, id: string): ApplicationDataType => {
   const data: ApplicationDataType = prepareData(reqBody, ManagingOfficerKeys);
   data[UsualResidentialAddressKey] = mapFieldsToDataObject(reqBody, UsualResidentialAddressKeys, AddressKeys);
   data[DateOfBirthKey] = mapFieldsToDataObject(reqBody, DateOfBirthKeys, InputDateKeys);
+  data[StartDateKey] = mapFieldsToDataObject(reqBody, StartDateKeys, InputDateKeys);
+  data[ResignedOnKey] = reqBody["is_still_mo"] === '0' ? mapFieldsToDataObject(reqBody, ResignedOnKeys, InputDateKeys) : {};
```

Comment:
I did this too but need to be careful these aren't set for registration journey.

```  // only set start_date and resigned_on keys for Update journey
  if (reqBody['start_date-day']){
    data[StartDateKey] = mapFieldsToDataObject(reqBody, StartDateKeys, InputDateKeys);
  }
  if (reqBody['is_still_mo']){
    data[ResignedOnDateKey] = reqBody["is_still_mo"] === '0' ? mapFieldsToDataObject(reqBody, ResignedOnDateKeys, InputDateKeys) : {};
  }```

Reply:
that's what I have in mine at the minute but let me know if you go with something else!

Reply:
Also just seeing that start_date out of scope for. your ticket so not sure you need any start_date related changes here at all

Reply:
what if reqBody['is_still_mo'] === 0 - you won't get to second then body. Use contains?